File: WebUI/ClientApp/src/app/core/entity/court/court.ts
import { Account } from "../account/account";

export class Court {

  public id: number;
  public name: string;
  public workingHours: string;
  public priceForHour: string;
  public phoneNumber: string;
  public dateOfCreation: Date;
  public createAt: Date;
  public modifyAt: Date;
  public rememberMe: boolean;
  public termAndCondition: boolean;

  public accountId: number;
  public account: Account = new Account();

  constructor() { }
}

File: Domain/Entities/Court.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Court
    {
        public int Id { get; set; }
        public int AccountId { get; set; }
        public string Name { get; set; }
        public string WorkingHours { get; set; }
        public string PriceForHour { get; set; }
        public string PhoneNumber { get; set; }
        public DateTime DateOfCreation { get; set; }
        public DateTime CreateAt { get; set; }
        public DateTime ModifyAt { get; set; }
        public Account Account { get; set; }
       
    }
}
